scripts/analyse-go-binary.py

```diff
@@ -0,0 +1,200 @@
+# Copyright (C) 2023 Mandiant, Inc. All Rights Reserved.
+
+import re
+import sys
+import struct
+import logging
+import argparse
+from typing import List, Iterable, Optional
+
+import pefile
+from tqdm import tqdm
+
+from floss.results import StaticString, StringEncoding
+
+logger = logging.getLogger(__name__)
+
+
+final_size = 0
+
+MIN_STR_LEN = 4
+
+
+def find_longest_string(bindata):
```

Comment:
I feel this is limiting us to a subset of all the strings. Why not look for all the strings (maybe in specific regions of the file like .data/.rdata) and calculate what we cover? Goal should be at least to recover 80% of all the strings in the file.